Index: src/O1_easy/O5_AddBinary_67.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/O1_easy/O5_AddBinary_67.java b/src/O1_easy/O5_AddBinary_67.java
new file mode 100644
--- /dev/null	
+++ b/src/O1_easy/O5_AddBinary_67.java	
@@ -0,0 +1,24 @@
+package O1_easy;
+
+import java.math.BigInteger;
+
+public class O5_AddBinary_67 {
+    public static void main(String[] args) {
+        Solution_67 solution_67 = new Solution_67();
+        System.out.println(solution_67.addBinary("11", "1"));
+    }
+}
+class Solution_67 {
+    public String addBinary(String a, String b) {
+
+        BigInteger p = new BigInteger(a, 2);
+        BigInteger q = new BigInteger(b, 2);
+        BigInteger r = p.add(q);
+        System.out.println(r);
+
+        return r.toString();
+
+    }
+}
+
+
Index: src/.~lock.All Solution At Once.docm#
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/.~lock.All Solution At Once.docm# b/src/.~lock.All Solution At Once.docm#
new file mode 100644
--- /dev/null	
+++ b/src/.~lock.All Solution At Once.docm#	
@@ -0,0 +1,1 @@
+,imtiaz,imtiaz,24.10.2022 21:34,file:///home/imtiaz/.config/libreoffice/4;
\ No newline at end of file
